src/Adapters/Auth/index.js

```diff
@@ -63,6 +63,8 @@ const providers = {
   ldap,
 };
 
+let authAdapters = {};
+
 // Indexed auth policies
```

Comment:
Bad pattern we should not introduce global singletons, parse is designed around dependency injection, and inversion of control, if you want to cache something at init time, check the https://github.com/parse-community/parse-server/pull/8510. You need to add an internal prop on the config.

Then this prop is copied (no memory footprint) between each request

Reply:
@mtrezza  @dblythy, i made some feedbacks

Reply:
Good feedback, I will resolve